src/configure-robot.ts

```diff
@@ -1,111 +1,21 @@
-import { Application } from "probot";
-import GithubAPI from "./config/github-api";
-import Redis from "ioredis";
-import RateLimit from "express-rate-limit";
-import RedisStore from "rate-limit-redis";
-import { createAppAuth } from "@octokit/auth-app";
-import { findPrivateKey } from "probot/lib/private-key";
-import getRedisInfo from "./config/redis-info";
+import {Application, Probot} from "probot";
 import setupFrontend from "./frontend";
 import setupGitHub from "./github";
-import statsd from "./config/statsd";
-import { isIp4InCidrs } from "./config/cidr-validator";
-import Logger from "bunyan";
-import { metricError } from "./config/metric-names";
-import logMiddleware from "./middleware/log-middleware";
-import { getLogger } from "./config/logger";
-import { NextFunction, Request, Response } from "express";
-import { booleanFlag, BooleanFlags } from "./config/feature-flags";
+import {overrideProbotLoggingMethods} from "./config/logger";
 
+export async function setupApp(app: Application): Promise<Application> {
+	setupGitHub(app);
+	setupFrontend(app);
 
-/**
- * Get the list of GitHub CIDRs from the /meta endpoint
- *
- * This is a weird one, the /meta endpoint has a very small rate limit
- * for non-authed users, but also doesn't allow queries from authenticated
- * apps, it must be from an installation of an application. So we fetch
- * the first installation and query as it. It only happens on boot, and
- * shouldn't affect anything else, it's theoretically safe.
- *
- * @returns {string[]} A list of CIDRs for GitHub webhooks
- */
-async function getGitHubCIDRs(logger: Logger): Promise<string[]> {
```

Comment:
This one is removed cause we are not expecting Frontend to be called form Github too often anyway, and this logic is not working for probot's managed endpoints